created a neo4j database, added readme. (#28)
TietoDigitalization/friends-tap: Neo4j/config/README.md

# NEO4j Database

> The database is built using a docker image hosted in the registry(https://registry.oso.it-partner.com) for the project. A custom config file is used since some settings for neo4j was not supported in the tanzu kubernetes cluster. Maybe after an upgrade to 1.5 we can use the default config file? To reaach the db locally use a kubectl port-forward command, kubectl has to set up to the tap cluster(see the instructions in the projects confluence page.) The command to connect is:

Reviewer: ```suggestion
The database is built using a docker image hosted in the registry(https://registry.oso.it-partner.com) for the project. A custom config file is used since some settings for neo4j were not supported in the tanzu kubernetes cluster. Maybe after an upgrade to 1.5 we can use the default config file? To reach the db locally use a kubectl port-forward command, kubectl has to set up to the tap cluster(see the instructions in the projects confluence page.) The command to connect is:
```

```
kubectl port-forward -n default neo4j-deployment-7ddf48c88f-4kjp5 7687:7687
```

> An image should be in the registry but if a new one should be built i t can be done by using the dockerfile and the following command(for building a linux image when you are using a mac with an arm64 architecture):

Reviewer: ```suggestion
An image should be in the registry but if a new one should be built it can be done by using the dockerfile and the following command(for building a linux image when you are using a mac with an arm64 architecture):
```
